modeling-cmds/src/def_enum.rs

```diff
@@ -960,6 +960,15 @@ define_modeling_cmd_enum! {
             pub parameters: Option<PerspectiveCameraParameters>,
         }
 
+        /// Fit the view to the specified object(s).
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct ZoomToFit {
+            /// Which objects to fit to
+            pub object_ids: Option<Vec<Uuid>>,
```

Comment:
What is the behaviour when this is None? What is the behaviour when this is Some(Vec![])?

If there's no difference, then just use a normal Vec no option. Either way, say in a comment so users know.